akka-stream/src/main/scala/akka/stream/scaladsl/Hub.scala

```diff
@@ -7,8 +7,8 @@ import java.util
 import java.util.concurrent.atomic.{ AtomicLong, AtomicReference }
 
 import akka.NotUsed
-import akka.dispatch.AbstractNodeQueue
-import akka.stream._
+import akka.dispatch.{ AbstractNodeQueue, ExecutionContexts }
+import akka.stream.{ StreamDetachedException, _ }
```

Comment:
what does such import mean?

Reply:
That's weird. IntelliJ did it.

Reply:
"Import everything, but especially StreamDetachedException" :D

Reply:
:rofl: